python/lsst/sims/movingObjects/chebyshevUtils.py

```diff
@@ -9,7 +9,7 @@
 
 # Evaluation routine.
 
-def chebeval(x, p, interval=(-1., 1.), doVelocity=True):
+def chebeval(x, p, interval=(-1., 1.), doVelocity=True, mask=False):
```

Comment:
I kind of thought the same thing .. this is Yusra's code, and tbh, I didn't have/want to take the time to delve in and figure it out in a better way. I agree that it seems like there ought to be a better way and we should put it in as a future ticket (even if it turns out not to be faster, it might be more readable?).